### PR TITLE
Feature/alternative file list

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
File list is now implemented as real list without pagination

## Motivation and Context
- Use all the screen space
- get rid of pagination

## How Has This Been Tested?
Manually

## Screenshots (if appropriate):
phoenix-file-list

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by felixheidecke** [comment]
@DeepDiver1975 what would you like me to review? I can't say too much about the code since I don't know about vuetify.

**Comment by DeepDiver1975** [comment]
Any Input is welcome. Thx

**Comment by scramb** [comment]
i like this attempt more than the v-data-table even if we're missing free functions like sorting

File: apps/files/src/components/FileUpload.vue [hunk 1]
```diff
@@ -81,7 +75,10 @@ export default {
 
     onProgress (e) {
       this.progress = parseInt(e.loaded * 100 / e.total)
-      this.$emit('progress', this.progress)
+      this.$emit('progress', {
+        fileName: 'bla',
```
[the remark this answers is not in the document]
Comment (DeepDiver1975) [reply apps/files/src/components/FileUpload.vue]: :see_no_evil:

**Pull request merged**